Graphics/GraphicsEngineD3D12/include/DescriptorHeap.hpp

```diff
@@ -341,23 +341,23 @@ class DescriptorHeapAllocationManager
     // this value may be different from m_HeapDesc.NumDescriptors
     Uint32 m_NumDescriptorsInAllocation = 0;
 
-    // Allocations manager used to handle descriptor allocations within the heap
-    std::mutex                     m_FreeBlockManagerMutex;
-    VariableSizeAllocationsManager m_FreeBlockManager;
-
-    // Strong reference to D3D12 descriptor heap object
-    CComPtr<ID3D12DescriptorHeap> m_pd3d12DescriptorHeap;
-
     // First CPU descriptor handle in the available descriptor range
     D3D12_CPU_DESCRIPTOR_HANDLE m_FirstCPUHandle = {0};
 
     // First GPU descriptor handle in the available descriptor range
     D3D12_GPU_DESCRIPTOR_HANDLE m_FirstGPUHandle = {0};
 
     size_t m_MaxAllocatedSize = 0;
+        
+    // Allocations manager used to handle descriptor allocations within the heap
```

Comment:
Please don't change the order of members - the order in which they are destructed may be important

Reply:
OK, will revert.